.changeset/eleven-beds-play.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-search-backend': patch
+'@backstage/plugin-catalog-backend': patch
+---
+
+Updated to use the improved OpenAPI Jest validation.
```

Comment:
Just an update to the schema in the search backend right? the rest is just test code that we can skip changesets for

Reply:
Yup, noticed that my initial schema update was incorrect so I've adjusted that as well.